spec/features/dataset_show_page_spec.rb

```diff
@@ -1,125 +1,107 @@
+      }]
+
+  describe 'Meta data' do
+    it 'displays a location if there is one' do
+      dataset = create_dataset(DATA_TITLE)
       index_and_visit(dataset)
-      expect(page).to have_content("Expected update: 24 July 2018")
+      expect(page).to have_content('Geographical area: London Southwark')
     end
+  end
 
-    it "monthly" do
-      dataset = create_dataset("Lovely data", "monthly", datafiles_w_enddate)
-      index_and_visit(dataset)
-      expect(page).to have_content("Expected update: 24 April 2018")
-    end
+  describe 'Datafiles are present' do
+    describe 'Data links'
 
-    it "daily" do
-      dataset = create_dataset("Lovely data", "daily", datafiles_w_enddate)
-      index_and_visit(dataset)
-      expect(page).to have_content("Expected update: 25 March 2018")
+    describe 'Additional info' do
+      it 'Is displayed if available' do
+        notes = 'Some very interesting notes'
+        dataset = create_dataset(DATA_TITLE, 'annual', DATA_FILES_WITH_ENDDATE, notes)
+        index_and_visit(dataset)
+        expect(page).to have_content(notes)
+      end
     end
 
-    it "never, one off" do
-      ["never", "one off"].each do |freq|
-        dataset = create_dataset("Lovely data", freq, datafiles_w_enddate)
+    describe 'Publisher' do
+      it 'Is displayed if available' do
+        dataset = create_dataset(DATA_TITLE, 'annual', DATA_FILES_WITH_ENDDATE)
         index_and_visit(dataset)
-        expect(page).to have_content("No future updates")
+        publisher = 'Ministry of Defence'
+        expect(page).to have_content(publisher)
       end
     end
 
-    it "discontinued" do
-      dataset = create_dataset("Lovely data", "discontinued", datafiles_w_enddate)
-      index_and_visit(dataset)
-      expect(page).to have_content("Dataset no longer updated")
+    describe 'Feedback' do
+      it 'Is displayed if available' do
+        dataset = create_dataset(DATA_TITLE, 'annual', DATA_FILES_WITH_ENDDATE)
+        index_and_visit(dataset)
+        expect(page).to have_content('Was this page useful for you?')
+      end
     end
   end
 
-  context "when a dataset has datafiles with no end dates" do
-
-    datafiles_no_enddate = [
-      {"id" => 1,
-       "name" => "I have no end date",
-       "url" => "https://good_data.co.uk",
-       "start_date" => "1/1/15",
-       "end_date" => nil,
-       "updated_at"=> "2016-08-31T14:40:57.528Z"
-     },
-      {"id" => 2,
-       "name" => "I have an end date",
-       "url" => "https://good_data.co.uk",
-       "start_date" => "1/1/15",
-       "end_date" => nil,
-       "updated_at"=> "2016-08-31T14:40:57.528Z"
-     }]
-
-     it "uses the updated_at field to calculate next update" do
-       dataset = create_dataset("Lovely data", "annual", datafiles_no_enddate)
-       index_and_visit(dataset)
-       expect(page).to have_content("Expected update: 31 August 2017")
-     end
-  end
+  describe 'Datafiles are not present' do
+    context 'Sections' do
```

Comment:
Convention is context is for state and describe is for things. I think they're the same method - but just fyi.